src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -1667,6 +1667,7 @@ public void execute()
       
       syncPublishPath(document.getPath());
       initStatusBar();
+      lintManager_.relintAfterDelay(5000);
```

Comment:
Does this change the way non-spellcheck lint behaves? We also probably want to make it this delay configurable via a e.g. a UI pref (no need to add actual UI for the pref at this point)

Reply:
It'll run the non-spellcheck lint on document load, yeah. In my testing nothing bad has seemed to come up from it. UI pref added.